src/components/Checkbox/Checkbox.scss

```diff
@@ -47,76 +32,108 @@
             display: flex;
             align-items: center;
         }
+
+        &:hover {
+            .Checkbox__box {
+                background: var(--color-background-selected-subtlest-hover, #CCE4FF);
+            }
+        }
+
+        &:active {
+            .Checkbox__box {
+                background: var(--color-background-selected-subtlest-pressed, #99CAFF);
+            }
+        }
     }
 
     &__svg {
         position: absolute;
-        top: 0;
+        top: var(--border-width-thick);
         left: 0;
         fill: none;
-        stroke: var(--color-background);
+        fill-rule:evenodd;
+        clip-rule:evenodd;
+        stroke: var(--color-background-information-subtlest-default);
         stroke-width: 2;
         stroke-linecap: round;
         stroke-linejoin: round;
     }
 
+    &--disabled {
+        .Checkbox__label {
+            .Checkbox__box {
+                background: var(--color-background-selected-subtlest-disabled, #E6F2FF);
+                border-color: var(--color-border-disabled, #CCC); 
+
+                &:focus {
+                    box-shadow: none;
+                }
+            }
+        }
+    }
+
     &__input {
-        display: none;
+        width: 0;
+        height: 0;
 
-        &--indeterminate {
```

Comment:
I also couldn't get mixed checkbox while testing... isn't indeterminate was responsible for it or am I missing something?
![Uploading Screenshot 2024-07-19 at 17.15.02.png…]()

Reply:
fixed the story

Reply:
works perfectly now :-) Thank you